boards/arm/nrf54l15dk_nrf54l15/revision.cmake

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# This is a list of all possible revisions
+set(revisions "soc1")
```

Comment:
@tejlmand this looks like the wrong way to check?

Reply:
It's acceptable when the revision in use is a free form text string.
Such cases must be implemented manually as there is no universal way to treat string.

Sure one can do a plain str compare, but how about greater / less than, for example: alpha, beta, gamma, ...